src/elements/checkbox/checkbox.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('semantic.ui.elements.checkbox', [])
+
+    .directive('smCheckbox', smCheckbox);
+
+  function smCheckbox() {
+    return {
+      restrict: 'E',
+      require: '?ngModel',
+      transclude: true,
+      scope: {
+        ngDisabled: '='
+      },
+      template: '<div class="ui checkbox" ng-class="{ checked: isChecked }">' +
```

Comment:
just to be clear, ng-transclude can be avoided by just manually transcluding --- but that might be an issue if an author tries to transclude an element transclusion directive. we've had some confusing ways of working around this in core, i'd have to do some research to figure out if anything special is needed for that. But at the end of the day, it would be really good to avoid creating new scopes for a small form control.